packages/ses/src/module-proxy.js

```diff
@@ -11,10 +11,21 @@
 // and eventually connecting it to to the proxied exports.
 
 import { makeAlias } from './module-load.js';
-import { create, freeze } from './commons.js';
+import {
+  Proxy,
+  TypeError,
+  create,
+  freeze,
+  ownKeys,
+  reflectGet,
+  reflectGetOwnPropertyDescriptor,
+  reflectHas,
+  reflectIsExtensible,
+  reflectPreventExtensions,
+} from './commons.js';
+import { assert } from './error/assert.js';
 
-// q, as in quote, for error messages.
-const q = JSON.stringify;
+const { quote: q } = assert;
 
```

Comment:
Another drive-by change. The mechanical change would have been to use `stringifyJson` from `commons.js`, but I’ve been meaning to refactor the lines of `const q = JSON.stringify` to use the new assert function anyway.